index.html

```diff
@@ -3,9 +3,13 @@
   <head>
     <meta charset="UTF-8">
     <title>First Webpage</title>
+    <link rel="stylesheet" href="index.css">
+    <script src="index.js"></script>
   </head>
   <body>
     <h1>First Webpage</h1>
     <p>Hello, world!</p>
+    <p>My favourite color is <span class="fav-color">pink</span>.</p>
+    <button name="click-me" onclick="click_me()">Click me</button>
```

Comment:
It is considered bad practice to use the `onclick` attribute. See http://stackoverflow.com/questions/5871640/why-is-using-onclick-in-html-a-bad-practice.

Instead, give the button an ID in the HTML and find it with JavaScript and set up a click handler that way instead. See https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementById and https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener.